### DISCUSSION TITLE
help me make this text white

### DISCUSSION THREAD

**Discussion by justageek** [romkatv/powerlevel10k#2778] [General]
![Screenshot 2024-10-18 at 10 41 18 AM](https://github.com/user-attachments/assets/f1cb9967-4a40-4965-b03d-170747c1c8e0)
the text for the branch icon and the branch name, I can't seem to get it to be white.

**Comment by gscalise** [comment]
Check for this section in `~/.p10k.zsh`, in the `my_git_formatter` function, and change to the color definitions you want (ie `%15F` for pure white).

```zsh
    if (( $1 )); then
      # Styling for up-to-date Git status.
      local       meta='%246F'  # grey foreground
      local      clean='%76F'   # green foreground
      local   modified='%178F'  # yellow foreground
      local  untracked='%39F'   # blue foreground
      local conflicted='%196F'  # red foreground
    else
      # Styling for incomplete and stale Git status.
      local       meta='%244F'  # grey foreground
      local      clean='%244F'  # grey foreground
      local   modified='%244F'  # grey foreground
      local  untracked='%244F'  # grey foreground
      local conflicted='%244F'  # grey foreground
    fi
```

if you just want to change the branch name, it's always printed out using the `clean` color, so you might be OK with just changing that one to `'%15F'`

**Reply by gscalise** [reply]
Also, it's probably showing grey because it's incomplete/stale (`gitstatus` is not able to get the repository status, you should check its logs)
